### ISSUE TITLE
Low SV Count When Performing SV Calling on HG002 ONT Data

### ISSUE THREAD

**Issue by Flooooooooooooower** [kcleal/dysgu#106]
Hello, when I use Dysgu for SV calling with ONT data, the number of SVs obtained is only 5000. Below is my code. Could you please help me understand why this is happening?

```
/dysgu call --pl nanopore --min-size 50 -c -p 20 --mode nanopore ./GCF_000001405.25_GRCh37.p13_genomic.fa ./HG002.ONT_vulcan_dysguwd ./HG002.ONT_vulcan.sort.bam
```

**Comment by kcleal** [comment]
Hi @Flooooooooooooower,
Could you share the stderr output from dysgu. I have not tested vulkan mapper, so not sure if it's compatible.

**Comment by kcleal** [comment]
Sorry, I meant the log output that is generated by dysgu e.g:

```
2024-08-19 15:44:55,602 [INFO   ]  [dysgu-run] Version: 1.6.6
2024-08-19 15:44:55,602 [INFO   ]  run --clean -x --procs 12 ucsc.hg19.fa wd HG002.bam
2024-08-19 15:44:55,603 [INFO   ]  Destination: wd
2024-08-19 15:45:20,409 [INFO   ]  dysgu fetch HG002.bam written to wd/HG002.dysgu_reads.bam, n=1394166, time=0:00:24 h:m:s
2024-08-19 15:45:20,409 [INFO   ]  Input file is: wd/HG002.dysgu_reads.bam
2024-08-19 15:45:20,433 [INFO   ]  Sample name: HG002.dysgu_reads
2024-08-19 15:45:20,433 [INFO   ]  Writing vcf to stdout
2024-08-19 15:45:20,433 [INFO   ]  Running pipeline
etc
```

Im not sure if you have this available for the different mappers, but If you do, it can be helpful for debugging and I might be able to offer some insight about what is going on.

I was wondering what type of ONT reads are these? Are they 'Kit14' or older R9.4 reads? Also what coverage do you have? Have you tested other SV callers?

Also might be worth updating to the latest version of dysgu v1.6.6.

**Comment by Flooooooooooooower** [comment]
Hello, the issue has been resolved when I used the latest version 1.6.6. Thanks for your help.
